docs/mongocli/command/mongocli-ops-manager-logs-keyProviders-list.txt

```diff
@@ -0,0 +1,71 @@
+.. _mongocli-ops-manager-logs-keyProviders-list:
+
+===========================================
+mongocli ops-manager logs keyProviders list
+===========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Lists all key provider configurations found in the encrypted log file.
```

Comment:
in the encrypted Ops Manager audit log file.

Reply:
updated without `Ops Manager`, same reasons as above